controllers/moderationController.js

```diff
@@ -0,0 +1,21 @@
+const { CheaterPhone, PhoneDescription } = require('../models');
+
+// READ (all categories)
```

Comment:
all categories?

Reply:
lol, nope) phones to moderate ofc) fixed